.github/workflows/arduino_packing.yml

```diff
@@ -0,0 +1,93 @@
+name: avrdude_packing
+
+env:
+  # The name of the project
+  PROJECT_NAME: avrdude
+  DIST_DIR: dist
+  ARTIFACT_NAME: dist
+
+on:
+  push:
```

Comment:
here you can specify a branch that will trigger the action (main in your case)

Reply:
Done. I also added the label part. Please help to check. Thanks.